packages/fast-application-utilities/README.md

```diff
@@ -1,19 +1,22 @@
 # FAST application utilities
-A list of any utilities commonly used by FAST applications.
+
+A list of utilities used by FAST applications.
 
 ## Localization
+
 The isRTL utility determines if a string which should correspond to a language or language-locale is right to left.
```

Comment:
A thought - do you think we should surround all function names/calls, enums, etc. with backticks? (i.e. `isRTL`)